hugo/layouts/partials/projects/projects.html

```diff
@@ -0,0 +1,24 @@
+<section class="projects band band_grey">
+    <div class="band__container">
+        {{ range $categoryName := .categories.order }}
+            {{ $category := index $.categories.contents $categoryName }}
+            {{ $title := $category.name }}
+            {{ $gradient := $category.colors }}
+            <h1>
+                {{ partial "svg" $categoryName }}
+                <span class="title">{{ $title }}</span>
+                <span id="{{ $categoryName }}" class="anchor"></span>
+            </h1>
```

Comment:
Am i losing something here or this produces:

```html
div
  h1
  div.cards
  h1
  div.cards
  h1
  div.cards
```

If it produces that html, please, fix it. No multiple `h1` in the same _section_